### PR TITLE
[DO-1466] Add link to pep.org.ua field

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
no need to add field to model, just add property and calculate url in runtime

**Comment by Iva-khar** [comment]
> no need to add field to model, just add property and calculate url in runtime

Done. Add to detail, list & short PEP serializers

File: business_register/models/pep_models.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ class Pep(DataOceanModel):
         (PEP_ASSOCIATED_PERSON, _("Associated person")),
         (PEP_FAMILY_MEMBER, _("Family member")),
     )
+    PEP_ORG_UA_PROFILE_URL = 'https://pep.org.ua/uk/person/'
```
Comment (roman-tiukh) [inline business_register/models/pep_models.py:38]: no need this var

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/models/pep_models.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/models/pep_models.py]: deleted

**Review by roman-tiukh — APPROVED** [review]

**Review by IegorSoboliev — APPROVED** [review]

**Pull request merged**